src/main/java/roomescape/global/config/RestClientConfig.java

```diff
@@ -0,0 +1,58 @@
+package roomescape.global.config;
+
+import org.apache.hc.client5.http.config.ConnectionConfig;
+import org.apache.hc.client5.http.config.RequestConfig;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager;
+import org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManagerBuilder;
+import org.apache.hc.core5.util.Timeout;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestClient;
+import org.springframework.web.client.RestClient.Builder;
+
+@Configuration
+public class RestClientConfig {
+
+    private Timeout CONNECTION_TIMEOUT = Timeout.ofSeconds(3);
+    private Timeout SOCKET_TIMEOUT = Timeout.ofSeconds(2);
+    private Timeout READ_TIMEOUT = Timeout.ofSeconds(30);
```

Comment:
```suggestion
    private final Timeout CONNECTION_TIMEOUT = Timeout.ofSeconds(3);
    private final Timeout SOCKET_TIMEOUT = Timeout.ofSeconds(2);
    private final Timeout READ_TIMEOUT = Timeout.ofSeconds(30);
```
이런 설정과 관련된 정보는 운영, 베타, 테스트환경 등과 같은 환경에 따라 달라질 수 있기 때문에 항상 고정하기보다는 profile별로 설정하기 위해서 `application.properties`에 넣어 가져다씁니다.